### ISSUE TITLE
Color Blocks shifting when using powerline enabled fonts

### ISSUE THREAD

**Issue by Cornelicorn** [microsoft/terminal#3085]
# Environment

```none
Microsoft Windows [Version 10.0.18362.388]
Windows Terminal (Preview) 0.5.2762.0

tmux zsh (bullet-train theme) on Ubuntu WSL 18.04.3 LTS
```

# Steps to reproduce

Install tmux or zsh and install a theme with color blocks (e.g.  [bullet-train theme](https://github.com/caiogondim/bullet-train.zsh)) and configure a powerline-font in Windows Terminal.

# Expected behavior
Font used for startup does not matter.

# Actual behavior

If you start zsh/tmux with a Powerline font, all color blocks are shifted slighty:
![image](https://user-images.githubusercontent.com/40914430/66259591-fdd06a00-e7b2-11e9-8861-0eb5f4c36c58.png)

If you start with a non-powerline font (e.g. Consolas) the blocks are the correct position:
![image](https://user-images.githubusercontent.com/40914430/66259619-48ea7d00-e7b3-11e9-8be9-2aa000b0effe.png)

Applying the powerline font after startup works as it should:
![image](https://user-images.githubusercontent.com/40914430/66259626-61f32e00-e7b3-11e9-99d1-00ef0969515e.png)

Using WSL without the Terminal App works (excpept lacking Emoji support)
![image](https://user-images.githubusercontent.com/40914430/66259710-87cd0280-e7b4-11e9-9817-47a203b7c109.png)

**Comment by Cornelicorn** [comment]
# Temporary fix:
save this script:
```bash
#!/bin/bash -e

# Replace the powerline font with the Consolas font
function shutdown {
sed -i '/"fontFace" : "YOUR_CUSTOM_FONT",/c\"fontFace" : "Consolas",' /mnt/YOUR_CUSTOM_PATH/profiles.json
}

trap shutdown EXIT #Execute function shutdown when terminal exits
#Start Tmux (change to bash/zsh/...)
tmux
```
And change the `commandline ` option in profiles.json to start it. (e.g. `"commandline" : "wsl.exe -d Ubuntu-18.04 /home/user/.zsh_startup",`

Then add ` sed -i '/"fontFace" : "Consolas",/c\"fontFace" : "YOUR_CUSTOM_FONT",' /mnt/YOUR_CUSTOM_PATH/profiles.json` at the end of the `rc` file of your shell (e.g. ~/.zshrc)

This way the font will be loaded after your shell is loaded and changed to the default before exit when closing Windows Terminal.

**Comment by Cornelicorn** [comment]
This fix unfortunately requires a tmux session running in the background

**Comment by DHowett-MSFT** [comment]
This is another version of /dup #633, fixed with #2066, in PR #2928.
